config.toml

```diff
@@ -57,6 +57,24 @@ enableRobotsTXT = true
     url = "/architecture"
     weight = "2"
     
+[[menu.main]]
+    name = "Platform Concept"
+    url = "/architecture/#platform-concept"
+    parent = "Architecture"
+    weight = "1"
+    
+[[menu.main]]
+    name = "Top-Level Architecture"
+    url = "/architecture/#top-level-architecture"
+    parent = "Architecture"
+    weight = "2"   
+
+[[menu.main]]
+    name = "Simulation Process from the User Perspective"
```

Comment:
This may appear very long and cause large bloating of the menu. I would consider a shorter name if possible

Reply:
@autumnfound We can rename "Simulation Process from the User Perspective"" into "Simulation Process". Would that work or do you mean also the other names?

Reply:
That would work! We just generally want to keep menu item names short, as that text needs to fit into the somewhat narrow drop-down menu. We have a "one-size fits all" approach for the drop-down menus, as configuring them to be different widths would cause a lot of unneeded complexity